pkg/web/api/v1/user.go

```diff
@@ -104,3 +109,46 @@ func (u *User) Model() (model *models.User, err error) {
 
 	return model, nil
 }
+
+func (u UserPassword) Validate() (err error) {
```

Comment:
Password strength validation is here … not sure if this is the right place for it or not.

Reply:
Would it be better to put it in `auth`?

Reply:
Yes, it would be better to put it into `auth`. I'll do that!